pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -113,6 +128,13 @@ stdenv.mkDerivation rec {
     # [2] https://sourceware.org/git/?p=glibc.git;a=blobdiff;f=io/bits/statx-generic.h;h=c34697e3c1fd79cddd60db294302e461ed8db6e2;hp=7a09e94be2abb92d2df612090c132e686a24d764;hb=88a2cf6c4bab6e94a65e9c0db8813709372e9180;hpb=c4e4b2e149705559d28b16a9b47ba2f6142d6a6c
     substituteInPlace meson.build \
       --replace 'has_statx = cc.links(statx_test)' 'has_statx = false'
+  '' + lib.optionalString stdenv.isDarwin ''
+    # Can likely be removed when updating derivation to 7.0
+    # Seems similar to https://github.com/containers/podman/issues/8016#issuecomment-1044945831
+    # If issue remains with 7.0, may need to be fixed upstream, or derivation may need other fix
+    # to stop setting CONFIG_LINUX.
+    sed -i "/fs_ss.add(when: 'CONFIG_LINUX', if_true: files('9p-util-linux.c'))/d" \
+        hw/9pfs/meson.build
```

Comment:
I don't much like this, but as long as it's temporary (until 7.0) I think it's fine.

Reply:
I don't like it either, but it's definitely temporary. If preferred, since this PR it's become a little [clearer](https://github.com/containers/podman/issues/13256#issuecomment-1054563197) that it's quite likely just an issue with the backport patch itself and the renamed file. I'm only 90% sure it would work, but I might try splitting the commit into two pieces (first rename, then add the comment) so it applies better. Homebrew is having the same issue. If that doesn't resolve it, I might still leave this in. I probably won't get to this until the weekend.

Reply:
I wouldn't worry about it too much.  Fix it if you want, but it's fine this way as a temporary measure. :)